cmd/dcrtime/dcrtime.go

```diff
@@ -169,23 +169,25 @@ func downloadV1(questions []string) error {
 		return err
 	}
 
-	if *debug {
-		fmt.Println(string(b))
-	}
-
 	// If this is a trial run return.
 	if *trial {
 		return nil
 	}
 
+	route := *host + v1.VerifyRoute
 	c := newClient(*skipVerify)
-	r, err := c.Post(*host+v1.VerifyRoute, "application/json",
+	r, err := c.Post(route, "application/json",
 		bytes.NewReader(b))
 	if err != nil {
 		return err
 	}
 	defer r.Body.Close()
 
+	if *debug {
```

Comment:
Move these back to before the request gets sent.  If the request is erroring the whole point of the debug statement is to be able to see what the url and req body are that are causing the error.

Reply:
Right makes sense, my bad. will move them

Reply:
Done mate @lukebp thanks for the review. 500 is also fixed by adding that missing return statement.